rust/src/io/object_store.rs

```diff
@@ -287,19 +292,19 @@ impl ObjectStore {
                 .await?,
                 scheme: String::from("s3"),
                 base_path: Path::from(url.path()),
-                block_size: 64 * 1024,
+                block_size: 256 * 1024,
```

Comment:
@eddyxu Was the 64k based on some benchmark? It seems like from the graph you had earlier, 256k didn't have much latency penalty vs 64k. And it seems like we are much more limited by request rate limits than by the block size when doing scans. But curious what other results we've seen.

Reply:
https://docs.google.com/spreadsheets/d/1eAyzJNQ8cL3ma-5zV5x9WaB3jtlR5rmgqnChwUW5tzM/edit#gid=0

Reply:
Yes, just chose 64K because it is lower. Let's do some benchmark again? These numbers were done on the C++ arrow.  Rust/tokio-io might be different.

Reply:
I've reverted this for now. We'll do some benchmarking in the near future.